gdk/gdk.go

```diff
@@ -2996,6 +2996,12 @@ func DragContextFromNative(l unsafe.Pointer) *DragContext {
 //-----------------------------------------------------------------------
 type Atom uintptr
 
+func AtomIntern(atom_name string, only_if_exists bool) Atom {
+	ptr := C.CString(atom_name)
+	defer cfree(ptr)
+	return Atom(C._gdk_atom_intern(gstring(ptr), gbool(only_if_exists)))
```

Comment:
Thank you. If it's possible to call without adding _gdk_atom_intern, you can write call of C.gdk_atom_intern directly.